src/components/views/messages/MessageActionBar.js

```diff
@@ -133,6 +148,37 @@ export default class MessageActionBar extends React.PureComponent {
         this.props.onFocusChange(focused);
     };
 
+    onKeyDown = (ev) => {
+        const {key} = ev;
+        const buttonNames = this.state.buttonNames;
+        const current = buttonNames.indexOf(this.state.focused);
+        let newIndex = null;
+
+        if (key === Key.ARROW_UP || key === Key.ARROW_DOWN) {
```

Comment:
Hmm, this seems to disagree with https://www.w3.org/TR/wai-aria-practices-1.1/examples/toolbar/toolbar.html